drvregistry/drvregistry.go

```diff
@@ -157,20 +149,6 @@ func (r *DrvRegistry) IPAMDefaultAddressSpaces(name string) (string, string, err
 	return i.defaultLocalAddressSpace, i.defaultGlobalAddressSpace, nil
 }
 
-func (r *DrvRegistry) initIPAMs(lDs, gDs interface{}) error {
```

Comment:
Ok, so the point is to allow the creator of `DrvRegistry` to control which ipam drivers needs to be registered.